java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -339,9 +340,65 @@ public interface StringBaseCommands {
      * @example
      *     <pre>{@code
      * // testKey1 = abcd, testKey2 = axcd
-     * Long result = client.lcs("testKey1", "testKey2").get();
+     * Long result = client.lcsLen("testKey1", "testKey2").get();
      * assert result.equals(3L);
      * }</pre>
      */
     CompletableFuture<Long> lcsLen(String key1, String key2);
+
+    /**
+     * Returns the indices and length of the longest common subsequence between strings stored at
+     * <code>key1</code> and <code>key2</code>.
+     *
+     * @since Redis 7.0 and above.
+     * @apiNote When in cluster mode, <code>key1</code> and <code>key2</code> must map to the same
+     *     hash slot.
+     * @see <a href="https://valkey.io/commands/lcs/">valkey.io</a> for details.
+     * @param key1 The key that stores the first string.
+     * @param key2 The key that stores the second string.
+     * @return A <code>Map&lt;String, Object&gt;</code> containing the indices of the longest common
+     *     subsequence between the 2 strings and the length of the longest common subsequence. The
+     *     <code>Object</code> mapped to the <code>"matches"</code> map key contains a two-dimensional
```

Comment:
I'd say this in an other words:
> The resulting map contains two keys: "matches" and "len":
> * "len" is mapped to the length of the common subsequence stored as `Long`.
> * "matches" mapped to a 3D array of .....

You need to describe what are these magic numbers in "matches". Maybe copy description from another client or source if you can find something. As an option you can refer to the example.